deploy/stacks/solution_bundling.py

```diff
@@ -14,18 +14,21 @@ def __init__(self, source_path=None):
 
     def try_bundle(self, output_dir: str, options: BundlingOptions) -> bool:
         requirements_path = Path(self.source_path, 'requirements.txt')
-        command = [
-            f'cp -a {self.source_path}/. {output_dir}/ && pip install -r {requirements_path} -t {output_dir}'
-        ]
-        subprocess.check_output(  # nosemgrep
-            command,  # nosemgrep
-            stderr=subprocess.STDOUT,  # nosemgrep
-            shell=True,  # nosec  # nosemgrep
+        subprocess.check_output(
+            ['cp', '-a', f'{self.source_path}/.', f'{output_dir}/'],
+            stderr=subprocess.STDOUT,
+            shell=False,
         )
 
-        ls_output = subprocess.check_output(  # nosemgrep
-            [f'ls -ll {output_dir}'],  # nosemgrep
-            stderr=subprocess.STDOUT,  # nosemgrep
-            shell=True,  # nosec  # nosemgrep
+        subprocess.check_output(
+            ['pip', 'install', '-r', requirements_path, '-t', output_dir],
+            stderr=subprocess.STDOUT,
+            shell=False,
+        )
+
+        ls_output = subprocess.check_output(
```

Comment:
do we need to store output in variable - does not seem like we are using `ls_output`